nixpkgs_review/review.py

```diff
@@ -579,6 +584,7 @@ def review_local_revision(
             build_graph=args.build_graph,
             nixpkgs_config=nixpkgs_config,
             extra_nixpkgs_config=args.extra_nixpkgs_config,
+            hook=args.hook,
```

Comment:
This Result class also needs this field. You can use `mypy nixpkgs_review` to test it for type errors like this.

Reply:
When I run it I get:

```
mypy nixpkgs_review
Success: no issues found in 20 source files
```

Not sure what a result class is in this context

Reply:
> When I run it I get:
> 
> ```
> mypy nixpkgs_review
> Success: no issues found in 20 source files
> ```
> 
> Not sure what a result class is in this context

Rebase the branch, before https://github.com/Mic92/nixpkgs-review/pull/371 a --strict was needed